jupyterlab_caip_optimizer/tests.py

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env python
```

Comment:
I recently removed these files from each subpackage in favor of a `test-python` command which calls a script hosted in the root of the repo.

https://github.com/GoogleCloudPlatform/jupyter-extensions/blob/39432248b5a82751c6f8de386177b7f74a625e38/jupyterlab_gcpscheduler/package.json#L35